addons/mass_mailing/models/mail_mail.py

```diff
@@ -81,17 +81,18 @@ def _prepare_outgoing_list(self, recipients_follower_status=None):
             view_url = self.mailing_id._get_view_url(email_to, self.res_id)
 
             # replace links in body
-            if not tools.is_html_empty(email_values['body']):
-                if f'{base_url}/unsubscribe_from_list' in email_values['body']:
-                    email_values['body'] = email_values['body'].replace(
-                        f'{base_url}/unsubscribe_from_list',
-                        unsubscribe_url,
-                    )
-                if f'{base_url}/view' in email_values['body']:
-                    email_values['body'] = email_values['body'].replace(
-                        f'{base_url}/view',
-                        view_url,
-                    )
+            for body in ['body', 'body_alternative']:
```

Comment:
Would rename `body` to `body_field`

Reply:
Why? It's confusing as they're not fields

Reply:
Fair enough ^^